mcp_proxy_for_aws/sigv4_helper.py

```diff
@@ -298,4 +298,4 @@ async def _inject_metadata_hook(metadata: Dict[str, Any], request: httpx.Request
 
         except (json.JSONDecodeError, KeyError, TypeError) as e:
             # Not a JSON request or invalid format, skip metadata injection
-            logger.error('Skipping metadata injection: %s', e)
+            logger.info('Skipping metadata injection: %s', e)
```

Comment:
Actually this should also be a `debug`
```suggestion
            logger.debug('Skipping metadata injection: %s', e)
```